manager/controllers/advisories.go

```diff
@@ -120,6 +120,10 @@ func buildAdvisoryAccountDataQuery(account int) *gorm.DB {
 		Where("sp.rh_account_id = ? ", account).
 		Group("sp.rh_account_id, sa.advisory_id")
 
+	if applyInventoryHosts {
```

Comment:
We already join with inventory during tag filtering. I think we shouldn't do it twice.

Reply:
Yes, but only if any tag is used otherwise the data is not joined with inventory.hosts using tag filtering. This ensures joining every time.